src/requests/process-paths.ts

```diff
@@ -61,6 +63,38 @@ function getSimpleName(url: string) {
   return method;
 }
 
+/**
+ * Returns name of the method
+ * @param method
+ */
 function getName(method: Method) {
   return _.upperFirst(_.camelCase(method.tags[0].replace(/(-rest)?-controller/, '')));
 }
+
+/**
+ * One of the allowed swagger formats is that under given url, there can be methods like get, post, put etc., but also
+ * parameters that often defines a path parameter common for the HTTP methods.
+ * This method extends HTTP method (get, post ...) parameters with the above mentioned parameters
+ * @param paths
+ */
+function preProcessPaths(paths: Paths): Paths {
+  Object.entries(paths).forEach(([pathKey, pathValue]) => {
+    if (pathValue.parameters) {
+      // deep copy is necessary
```

Comment:
Really? If so, could you, please, state the reason?

Reply:
Please have a look at the updated comment:

deep copy is necessary - the common parameters must be deleted so that when iterating over http methods, the common parameters will not be considered as one of them. Yet, the common parameters must be stored in separate variable (methodsParameters) for further use

Reply:
@fazpu, please, let's remove this one and merge. It's imo the last blocker. Thx